fabric-api-base/src/main/java/net/fabricmc/fabric/api/event/EventFactory.java

```diff
@@ -138,4 +119,24 @@ public static <T> Event<T> createWithPhases(Class<? super T> type, Function<T[],
 	public static String getHandlerName(Object handler) {
```

Comment:
I think we should also deprecate `getHandlerName`

Reply:
I didn't even know this was a thing...